src/Models/Job.ts

```diff
@@ -27,17 +38,21 @@ interface JobLength {
 export interface Job {
   title: string;
   description: string;
-  company: string;
-  location: string;
+  isCompanyListing: boolean;
+  employer: string;
```

Comment:
should this be a Reference to employer?

Reply:
I see it is included in the Schema def, not sure if Employer is needed then for type